src/pages/en/guides/content-collections.mdx

```diff
@@ -144,18 +144,18 @@ You can specify each expected property in the `schema` field of `defineCollectio
 import { z, defineCollection } from 'astro:content';
```

Comment:
Line 140 above might need updating too?

> You can specify each expected property in the `schema` field of `defineCollection`:

Could become something like?

> You can specify the expected shape of your frontmatter with the `schema` field of `defineCollection`:

Reply:
I like it!